### DISCUSSION TITLE
"VS Code does not generate affiliations when I change the format to PDF."

### DISCUSSION THREAD

**Discussion by diegourreamendez** [quarto-dev/quarto-cli#10133] [Q&A]
### Description

I am following the Publishing Quarto Manuscripts tutorial, and everything works perfectly using VS Code. However, when I change the format to PDF by adding 'format: pdf' at the end of the code, the names appear, but the affiliations do not. I would like to know if I need to modify something in the YAML code to include the complete information in my PDF. Here is the specific information I have in my session. It would be helpful if you could add something about this topic in this section.

Rendering PDF
running xelatex - 1
  This is XeTeX, Version 3.141592653-2.6-0.999996 (MiKTeX 24.4) (preloaded format=xelatex.fmt)
   restricted \write18 enabled.
  entering extended mode

running xelatex - 2
  This is XeTeX, Version 3.141592653-2.6-0.999996 (MiKTeX 24.4) (preloaded format=xelatex.fmt)
   restricted \write18 enabled.
  entering extended mode

Output created: articulo.pdf

Watching files for changes

Executing 'articulo.quarto_ipynb'

pandoc 
  to: latex
  output-file: articulo.tex
  standalone: true
  pdf-engine: xelatex
  variables:
    graphics: true
    tables: true
  default-image-extension: pdf
  number-sections: true

metadata
  documentclass: scrartcl
  classoption:
    - DIV=11
    - numbers=noendperiod
  papersize: letter
  header-includes:
    - '\KOMAoption{captions}{tableheading}'
  block-headings: true
  title: '...'
  author:
    - name: ...
      orcid: ...
      corresponding: false
      email: ...
      roles: []
      affiliations:
        - ...
    - name: ...
      email: ...
      orcid: ...
      corresponding: true
      roles: []
      affiliations:
        - ...
  abstract: |
    ....
  plain-language-summary: |
    ...
  key-points:
    - ...
    - ...
  date: last-modified
  bibliography:
    - ...
  citation:
    container-title: Earth and Space Science
  jupyter: python3

Rendering PDF
running xelatex - 1
  This is XeTeX, Version 3.141592653-2.6-0.999996 (MiKTeX 24.4) (preloaded format=xelatex.fmt)
   restricted \write18 enabled.
  entering extended mode

running xelatex - 2
  This is XeTeX, Version 3.141592653-2.6-0.999996 (MiKTeX 24.4) (preloaded format=xelatex.fmt)
   restricted \write18 enabled.
  entering extended mode

**Comment by mcanouil** [comment]
Could you properly format your post using code blocks for code and terminal outputs? Thanks.
If your code contains code blocks, you need to enclose it using more backticks, *i.e.*, usually four ` ```` `.

This being said, PDF follow the template. The default does not have affiliations, but Journal templates have it (https://quarto.org/docs/journals/).

- https://quarto.org/docs/extensions/listing-journals.html
- https://m.canouil.dev/quarto-extensions/extensions.html
